platform-core/platform-telemetry/pom.xml

```diff
@@ -34,6 +34,15 @@
 	<build>
 		<pluginManagement>
 			<plugins>
+				<plugin>
+					<artifactId>maven-assembly-plugin</artifactId>
+					<version>3.3.0</version>
+					<configuration>
+						<descriptors>
+							<descriptor>src/assembly/bin.xml</descriptor>
+						</descriptors>
+					</configuration>
+				</plugin>
```

Comment:
what is this change @rheafernandes

Reply:
I wanted to if that helps to show the telemetry coverage in circle ci